src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -5,20 +5,22 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:viewModels="clr-namespace:Dynamo.ViewModels"
              xmlns:ui="clr-namespace:Dynamo.UI"
+             xmlns:controls="clr-namespace:Dynamo.Controls"
              mc:Ignorable="d"
-             d:DesignHeight="161"
-             d:DesignWidth="410"
+             Width="300"
+             Height="300"
```

Comment:
Would this change affect default watch node size?

Reply:
Yes, before this change we would have a maximum height of 300 but now we don't want to put any limitation on that. 
So when we remove that, the height of the watch node would be dependent on the number of elements in the list and it would become much longer. So now we will have a default width and height same as the size of watch3d node and it can be expanded with the thumb icon.

Reply:
hmm - I think it would be nice if the default stayed the same as it was - so that existing graphs do not change in appearance when opened.

Reply:
So whats happening is, previously there was no height and width set at runtime for the watch node. Only the maximum height of 300 was set. So the height of the node would depend on the number of elements but once the max height is reached, the scroll bar appears. Now we don't have any max height set to the watch node and if no default height or width is set explicitly, then the watch node will be too long in case where input has a lot of elements. To avoid it, I thought we could make it consistent with the watch3d node.